app/src/main/java/com/github/se/cyrcle/model/map/MapViewModel.kt

```diff
@@ -142,17 +157,31 @@ class MapViewModel : ViewModel() {
     return Pair(bottomLeftCorner, topRightCorner)
   }
 
+  private val handler = Handler(Looper.getMainLooper())
+  private var lastUpdateTime = 0L
+  private val updateInterval = 1000L // 1 second
```

Comment:
For this value, there is a trade off between debouncing but making sure the app stays functionnal and display relevant infos concerning distances. Do you think 1 seconds is enough or maybe too much ? (Just a debate here). I personnally think 0.5 seconds could be better but that is just a minor detail.

Reply:
I will go with half a second. I don't think it matters so much in this screen. In my opinion, for the list screen we could even update once a minute / not update it. The user won't use it to refer themselves to a parking. Just to have a quick and easy overview